docs/book/misc/faq.md

```diff
@@ -0,0 +1,68 @@
+---
+description: We answer the most frequently asked questions about ZenML here
+---
+
+# FAQ
+
+#### How does ZenML compare to orchestrators (Airflow, Kubeflow, Flyte, etc.)
+
+An orchestrator in MLOps is the system component that is responsible for executing and managing the execution of a ML pipeline. ZenML pipelines can be run on multiple orchestration systems. There are [standard orchestrators](../extending-zenml/orchestrator.md) that ZenML supports out-of-the-box, but you are encourage to write your own [orchestrator](../extending-zenml/orchestrator.md) in order to gain more control as to exactly how your pipelines are executed!
+
+#### Can I use tool X? How does Y tool integrate with ZenML?
+
+Take a look at our [examples](https://github.com/zenml-io/zenml/tree/main/examples) directory, which showcases detailed example for each integration ZenML supports out-of-the-box.
+
+The ZenML team and community is constantly working to include more tools and integrations to the above list (check the [roadmap](https://zenml.io/roadmap) for more details). You can [upvote features](https://zenml.io/discussion) you'd like and add more ideas on the roadmap.
+
+#### How can I make ZenML work with my custom tool? How can I extend or build on ZenML?
+
+Depends on the tool, and its MLOps category. We have a full guide on this over [here](../extending-zenml/)!
+
+#### Why did you build ZenML?
+
+We built it because we scratched our own itch while deploying multiple ML models in production for the last 3 years. Our team struggled to find a simple yet production-ready solution whilst developing large scale ML pipelines, and built a solution for it that we are now proud to share with all of you!
```

Comment:
```suggestion
We built it because we scratched our own itch while deploying multiple machine learning models in production for the past three years. Our team struggled to find a simple yet production-ready solution whilst developing large-scale ML pipelines, and built a solution for it that we are now proud to share with all of you!
```